package.json

```diff
@@ -5,7 +5,6 @@
   "main": "index.js",
   "browserify": {
     "transform": [
-      "css-img-datauri-stream",
```

Comment:
this library has not been updated in 5 years and depends on a version of `mime` with a security vulnerability.  The only place it's used is to, at  build-time, generate a data uri for a single image.  So, I manually generated the data uri for that image and updated the css with it, and now it no longer needs the lib.